tests/functional/lang/eval-okay-inherit-from.exp

```diff
@@ -0,0 +1 @@
+[ 1 2 { __overrides = { y = { d = [ ]; }; }; c = [ ]; d = 4; x = { c = [ ]; }; y = «repeated»; } ]
```

Comment:
```
y = Â«repeatedÂ»;
```
Mojibake?

Reply:
I'm not sure what's going on with the Unicode rendering here. Seems to be messed up but only in the Files tab. I checked out the PR locally and the encoding is correct there. It looks like the files are being decoded as [Latin-1](https://en.wikipedia.org/wiki/ISO/IEC_8859-1) for some reason:

```
$ python3
>>> "«repeated»".encode("utf-8").decode("latin-1")
'Â«repeatedÂ»'
```

Maybe the literal `\r` in `tests/functional/lang/eval-fail-eol-2.nix` is tripping GitHub up?